packages/pretty-format/src/index.ts

```diff
@@ -520,4 +522,4 @@ prettyFormat.plugins = {
   ReactTestComponent,
 };
 
-export = prettyFormat;
+export default prettyFormat;
```

Comment:
this is a breaking change - it needs to stay `export =`.

If you need types, move the types to `@types/jest` and we'll move them back in together with making it ESM in a future major version 🙂